src/GitHub.InlineReviews/Views/CommentView.xaml

```diff
@@ -67,7 +67,7 @@
                         CornerRadius="3"
                         Padding="2 1"
                         Visibility="{Binding IsPending, Converter={ui:BooleanToVisibilityConverter}, FallbackValue=Collapsed}">
-                    <TextBlock FontSize="10">Pending</TextBlock>
+                        <TextBlock FontSize="10" Text="{x:Static prop:Resources.Pending}"></TextBlock>
```

Comment:
Now that the `<TextBlock>` doesn't have any content, it can use an auto-closing tag, i.e.:

```
<TextBlock FontSize="10" Text="{x:Static prop:Resources.Pending}"/>
```

This can be done in a lot of places, I won't add a comment for each one ;)